libs/sdk-ui-kit/src/Button/Button.tsx

```diff
@@ -24,26 +24,49 @@ export class Button extends React.Component<IButtonProps> {
     public buttonNode: HTMLElement;
 
     public render() {
-        const { id, tagName, title, disabled, tabIndex, type, iconLeft, iconRight, accessibilityConfig } =
-            this.props;
-        const { ariaLabel, ariaLabelledBy } = accessibilityConfig ?? {};
+        const {
+            id,
+            tagName,
+            title,
+            disabled,
+            tabIndex,
+            type,
+            iconLeft,
+            iconRight,
+            ariaLabel,
+            accessibilityConfig,
+            buttonRef,
+        } = this.props;
+        const {
+            isExpanded,
+            popupId,
+            ariaLabel: ariaLabelFromConfig,
+            ariaLabelledBy,
+        } = accessibilityConfig ?? {};
         const TagName = tagName as any;
         const effectiveValue = this.getEffectiveValue();
+        const ariaLabelToUse = ariaLabelFromConfig ?? ariaLabel;
 
         return (
             <TagName
                 id={id}
                 ref={(ref: HTMLElement) => {
                     this.buttonNode = ref;
+                    if (buttonRef) {
+                        buttonRef.current = ref;
+                    }
                 }}
-                title={title}
+                title={title || ariaLabelToUse}
                 className={this.getClassnames()}
                 type={type}
                 onClick={this._onClick}
                 tabIndex={tabIndex}
                 aria-disabled={disabled}
-                aria-label={ariaLabel}
+                aria-label={ariaLabelToUse}
                 aria-labelledby={ariaLabelledBy}
+                aria-controls={popupId}
+                aria-haspopup={!!popupId}
```

Comment:
Is `false` ok? Wouldn't it be better to not have it on the tag? How about `aria-expanded` and `aria-controls`? Doesn't it pollute the DOM when button is not used as an dropdown button?

Reply:
conditional props added